{"name": "Попытка входа с невалидными данными: неверный логин", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('courier_client', <FixtureRequest for <Function test_login_negative_cases[\\u043d\\u0435\\u0432\\u0435\\u0440\\u043d\\u044b\\u0439 \\u043b\\u043e\\u0433\\u0438\\u043d-<lambda>-404-\\u041d\\u0435 \\u0443\\u0447\\u0435\\u0442\\u043d\\u0430\\u044f \\u0437\\u0430\\u043f\\u0438\\u0441\\u044c \\u043d\\u0435 \\u043d\\u0430\\u0439\\u0434\\u0435\\u043d\\u0430]>>)", "trace": "file C:\\Users\\Fox Funny\\Sprint-7\\tests\\test_courier_login.py, line 22\n      @allure.story('Неуспешная авторизация')\n      @allure.title('Попытка входа с невалидными данными: {test_case}')\n      @pytest.mark.parametrize('test_case, credentials, expected_status, expected_message', [\n          ('неверный пароль',\n           lambda x: {\"login\": x[\"login\"], \"password\": \"invalid_password\"},\n           404, \"Не учетная запись не найдена\"),\n          ('неверный логин',\n           lambda x: {\"login\": \"invalid_login\", \"password\": x[\"password\"]},\n           404, \"Не учетная запись не найдена\"),\n          ('случайные логин и пароль',\n           lambda x: {\"login\": \"invalid_login\", \"password\": \"invalid_password\"},\n           404, \"Не учетная запись не найдена\"),\n          ('отсутствует логин',\n           lambda x: {\"login\": \"\", \"password\": x[\"password\"]},\n           400, \"Недостаточно данных для входа\"),\n          ('отсутствует пароль',\n           lambda x: {\"login\": x[\"login\"], \"password\": \"\"},\n           400, \"Недостаточно данных для входа\")\n      ])\n      def test_login_negative_cases(self, create_courier, test_case, credentials, expected_status, expected_message, courier_client):\nE       fixture 'courier_client' not found\n>       available fixtures: _session_faker, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, capteesys, create_courier, doctest_namespace, faker, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\nC:\\Users\\Fox Funny\\Sprint-7\\tests\\test_courier_login.py:22"}, "parameters": [{"name": "test_case", "value": "'неверный логин'"}, {"name": "credentials", "value": "<function TestCourierLogin.<lambda> at 0x0000021E5626D1C0>"}, {"name": "expected_status", "value": "404"}, {"name": "expected_message", "value": "'Не учетная запись не найдена'"}], "start": 1758566290574, "stop": 1758566290574, "uuid": "1f3ce2c7-d5f1-4c73-bbf7-ff81b25c3404", "historyId": "421ab3cf40e1f7201579d58b7f46298b", "testCaseId": "505bba1c9113ea857d47e178537c8295", "fullName": "tests.test_courier_login.TestCourierLogin#test_login_negative_cases", "labels": [{"name": "story", "value": "Неуспешная авторизация"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "What-Is-Lowe"}, {"name": "thread", "value": "5924-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}], "titlePath": ["tests", "test_courier_login.py", "TestCourierLogin"]}